### DISCUSSION TITLE
Inelastic Rope Simulation in Brax

### DISCUSSION THREAD

**Discussion by jc-bao** [google/brax#406] [General]
Hello Brax team,

I'm interested in simulating an inelastic rope within Brax, similar to [this example](https://mujoco.readthedocs.io/en/stable/_static/example.mp4). When the rope transitions from a loose to taut state, it seems to mimic an inelastic collision. Given Brax's capabilities, I believe this might be feasible. Could you provide any insights or guidance on this?

<img width="160" alt="image" src="https://github.com/google/brax/assets/60093981/eb10842a-709e-4203-8d61-aff0d3921bef">

**Comment by btaba** [comment]
Hi @jc-bao , it looks like the example you linked to is using tendons in MuJoCo, so I would take a look at the implementation here https://github.com/google-deepmind/mujoco. I'm not very familiar with the implementation but hope that helps

**Reply by jc-bao** [reply]
Yeah, a system with a tendon is a hybrid system, which shares a lot of similarities with the contact model. Issac also supports tendon now. It would be great if Brax could support it.

**Reply by btaba** [reply]
Check out our recent announcement: https://github.com/google/brax/discussions/409

tendons will more easily be supported through MJX, and subsequently a MJX brax backend

**Reply by jc-bao** [reply]
I believe my concern has now been alleviated. A big thank you to the Brax team for integrating Mujoco with the powerful Jax! 👏🙌
